frontend/src/app/oe/oe.routing.modules.ts

```diff
@@ -47,22 +47,6 @@ const routes: Routes = [
             path: 'blockrate-detail/:from/:to',
             component: EnergyDetailComponent,
           },
-          {
-            path: 'blockrate-summary',
-            component: EnergySummaryComponent,
-          },
-          {
-            path: 'blockrate-strike-detail',
-            component: StrikeDetailComponent,
-          },
-          {
-            path: 'blockrate-strike-summary',
-            component: StrikeSummaryComponent,
-          },
-          {
-            path: 'blockrate-strike-summary-with-guess',
-            component: StrikeSummaryWithGuessComponent,
-          },
```

Comment:
I see, that 'blockrate*' paths had been removed from routing, but I don't see how they will be handled instead? Will they be handled by another route?